pkg/plugin/processor/builtin/values.go

```diff
@@ -0,0 +1,35 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package builtin
+
+import "github.com/conduitio/conduit/pkg/foundation/cerrors"
+
+// references
+
+const (
```

Comment:
any suggestion of where to keep these values? or maybe another file name? lemme know what you think

Reply:
Maybe split the file into references.go and errors.go? I'm sure we'll have more commonly used errors.